docs/user-guide/installation/index.rst

```diff
@@ -0,0 +1,124 @@
+Installation
+============
+
+Pulp operates with three main components that get installed potentially on different
+machines.
+
+Server
+  This is the main application server that stores data and distributes content.
+
+Agent
+  This component runs on consumers and communicates with the server to provide remote content
+  management.
+
+Client
+  This is a command line component that comes as two pieces: admin-client,
```

Comment:
This could be more clearly stated that it is a management CLI. That could be stated more up front.